cx_Freeze/samples/pytz/test_pytz.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+from tzlocal import get_localzone
+
+RFC1123 = '%a, %d %b %Y %H:%M:%S %z'
+
+try:
+    tz = get_localzone()
+except:
+    import pytz
+    tz = pytz.utc
+    print("WARNING: fail to load pytz timezones")
+finally:
+    print(datetime.now(tz).strftime(RFC1123))
```

Comment:
tzlocal.get_localzone() simply returns "local" when the timzone information cannot be loaded -- but it still "works". :-) My suggestion is to use something like this instead:

```
import datetime
import pytz
import tzlocal

RFC1123 = '%a, %d %b %Y %H:%M:%S %z'

localTZ = tzlocal.get_localzone()
print("Local TZ:", localTZ)

otherTZ = pytz.timezone("US/Eastern")
print("Other TZ:", otherTZ)

print("Local Time:", datetime.datetime.now(localTZ).strftime(RFC1123))
print("Other Time:", datetime.datetime.now(otherTZ).strftime(RFC1123))
```

This yields an exception when frozen. What do you think?

Reply:
My sample is to prove that is failing to load pytz zoneinfo, not as example of use.

Reply:
localTZ = tzlocal.get_localzone()
print("Local TZ:", localTZ)
raises a exception, "don't still works" without the my patch or your patch when frozen.

Reply:
I think the issue is that the implementation of tzlocal that I am using (or the fact that I am using Fedora 31) causes the script you provided to "work" for me whereas it doesn't work for you. But if my suggested script fails for both of us, why not use it or something like it?